warehouse/email/ses/models.py

```diff
@@ -230,18 +233,17 @@ def _get_email(self):
 class EmailMessage(db.Model):
     __tablename__ = "ses_emails"
 
-    created = mapped_column(DateTime, nullable=False, server_default=sql.func.now())
-    status = mapped_column(
+    created: Mapped[datetime] = mapped_column(server_default=sql.func.now())
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏I'm already thinking we can create another custom type, since this is a repeat pattern we have:

```python
datetime_now = Annotated[datetime, mapped_column(server_default=sql.func.now())]
```

Reply:
https://github.com/pypi/warehouse/pull/14270/commits/4f39a6802c91ec63137570cfe0c8db7f4cb4733f